webapp/src/app.tsx

```diff
@@ -0,0 +1,16 @@
+import {loadConfig} from '../src/actions';
+import React, {useEffect} from 'react';
+import {useDispatch} from 'react-redux';
+
+const GetConfig = (): JSX.Element => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(loadConfig());
+    }, []);
+
+    // This container is used just for making the API call for fetching the config, it doesn't render anything.
+    return <></>;
+};
```

Comment:
```suggestion
const GetConfig = (): React.ReactNode => {
    const dispatch = useDispatch();

    useEffect(() => {
        dispatch(loadConfig());
    }, []);

    // This container is used just for making the API call for fetching the config, it doesn't render anything.
    return null;
};
```
try this once

Reply:
copied this from service now and azure .Is there any difference between the two ?